app/views/devise/unlocks/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend unlock instructions</h2>
```

Comment:
We are not sending emails for people to unlock their accounts so I think you can get rid of this.